refactor (#330)
KxSystems/docs: docs/wp/permissions/index.md

{: .wp-author}

> kdb+ processes often contain sensitive, proprietary information in the form of data or proprietary code and so it is important to restrict who can and cannot access this information. 

Reviewer: ```suggestion
kdb+ processes often contain sensitive, proprietary information in the form of data or proprietary code. Thus, it is important to restrict the access to this information. 
```
